syncd/syncd_hard_reinit.cpp

```diff
@@ -351,14 +352,14 @@ sai_object_id_t processSingleVid(sai_object_id_t vid)
     }
```

Comment:
what about SAI_OBJECT_TYPE_PRIORITY_GROUP and SAI_OBJECT_TYPE_QUEUE? also use lambda